fizzBuzzRedux.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Fizz Buzz Redux</title>
+</head>
+<body>
+  <form id="inputForm" action="javascript:onButtonClick()">
```

Comment:
Additionally, given the assignment constraints, were this a test for a job application that I was administering, this would disqualify you.